### ISSUE TITLE
Create survey for v2 API

### ISSUE THREAD

**Issue by jimchamp** [internetarchive/openlibrary#7576]
Before creating a V2 API for Open Library, we should first understand what developers want from an updated API.  A survey will help us determine what changes are wanted.

- [ ] Create short survey to help determine what people want from a new Openlibrary API
- [ ] Open survey to developers for some period of time
- [ ] Synthesize survey results

**Comment by brierjon** [comment]
One feature request #7629

**Comment by tfmorris** [comment]
If this is being motivated by complaints about the current API, it would be useful to know what those complaints are.

Also, as an application developer I'm *at least* as concerned about the data that I can access through the API as the form of the API, so it might be worth including some questions about data fields, data quality, etc. A stellar API to crap data doesn't allow one to build very compelling applications.

**Comment by mekarpeles** [comment]
I agree with @tfmorris. I also acknowledge we now have a powerful Edition-Search API that likely achieves many of patron's needs.

There's still confusion about what APIs to use, when, and also we may benefit from clearer documentation and feedback from the community about their use cases and direct feedback about what is working v. not working for them.

Let's bring this up as a topic during our Design Call and come up with a very simple survey we can put on the banner for 1 week.

**Comment by mekarpeles** [comment]
@jimchamp we may want to link this to #7452
